test/kubemark/resources/start-kubemark-master.sh

```diff
@@ -301,10 +301,13 @@ function prepare-log-file {
 function setup-addon-manifests {
   local -r src_dir="${KUBE_ROOT}/$2"
   local -r dst_dir="/etc/kubernetes/$1/$2"
+	local files=""
```

Comment:
whitespace, I nearly missed this
```suggestion
    local files=""
```

Reply:
Yeah, it presented itself fine in vscode, noticed it too when it was pushed up -_- fixed it in vim and repushed.